addons/website/static/src/components/configurator/configurator.js

```diff
@@ -206,13 +209,34 @@ class PaletteSelectionScreen extends Component {
         if (logoSelectInput.files.length === 1) {
             const file = logoSelectInput.files[0];
             const data = await utils.getDataURLFromFile(file);
-            this.dispatch('changeLogo', data);
-            this.updatePalettes();
+            const attachment = await this.rpc({
+                route: '/web_editor/attachment/add_data',
+                params: {
+                    name: 'temporary_logo',
+                    data: data.split(',')[1],
+                    res_model: 'website',
+                    res_id: session.website_id,
+                    is_image: true,
+                }
+            });
+            if (!attachment.error) {
+                this.dispatch('changeLogo', data, attachment.id);
+                this.updatePalettes();
+            } else {
+                this.notification.notify({
+                    title: _t(file.name),
```

Comment:
1) Don't translate file names? :thinking: 
2) _t is expecting a string between quotes, otherwise the term is not translatable (how could we guess what to translate?)
```javascript
_t("Hello"); // Translated !
```
```javascript
const variable = "Hello";
_t(variable); // Not translated !
```

Reply:
Yep quite stupid...